boot.php

```diff
@@ -1022,13 +1031,13 @@ function get_cached_avatar_image($avatar_image){
 
 		// The following code is deactivated. It doesn't seem to make any sense and it slows down the system.
```

Comment:
Cleanup: Can you please remove commented out code since nothing is planned to restore it.